### DISCUSSION TITLE
質問に対する回答後の編集について

### DISCUSSION THREAD

**Discussion by mariori12** [increments/qiita-discussions#616] [要望・アイデア・不具合]
日頃からお世話になっております。

Q&Aの回答ページについて
挙動に疑問を抱きましたので問い合わせいたします。

一次回答後、解答欄を編集時に
「Spaceキー」押下若しくは、
「Enter」キー押下にて「空白」若しくは「空行」を挿入しようとしたところ
画面下部の「回答する」欄にパークしました。
動作としては正常なのでしょうか。

環境は下記です
　・Windows10 KB5034441 迄 アップデート済み(現状)
　・Chrome：バージョン: 121.0.6167.161（Official Build） （64 ビット）

**Comment by tanaka-asami415** [comment]
@mariori12 
ご報告ありがとうございます。

弊社側でも試してみたのですが、同症状は再現しませんでした。
もしかしたら使用されているChromeの拡張機能の影響があるかもしれません。

もしも、拡張機能をオフにしていただいても同症状が出る場合は、
お手数をおかけしますが、動画を撮っていただけないでしょうか？🙏

**Reply by mariori12** [reply]
@tanaka-asami415 
ご確認いただきありがとうございました。

拡張機能の影響による現象でした。
お手を煩わせてしまい申し訳ございませんでした。

貴重なお時間の中調査いただきましたこと、重ねてお礼申し上げます。

**Reply by tanaka-asami415** [reply]
@mariori12 
とんでもないです。解決されてよかったです✨
また何かありましたら、Discussions等にご報告ください！
